web/src/hotkey.js

```diff
@@ -310,9 +310,7 @@ export function process_escape_key(e) {
 
             // Check for errors in compose box; close errors if they exist
             if ($(".compose_banner").length) {
-                compose_banner.clear_message_sent_banners();
-                compose_banner.clear_errors();
-                compose_banner.clear_warnings();
+                compose_banner.clear_all();
```

Comment:
This hunk makes sense to be in the first commit -- logically, the idea is that we're changing this have the same semantics but better described conceptually.

Reply:
Right, I'd forgotten to move this to the older commit, fixed.